sway/sway.desktop

```diff
@@ -3,3 +3,4 @@ Name=Sway
 Comment=An i3-compatible Wayland compositor
 Exec=sway-run.sh
 Type=Application
+DesktopNames=sway;wlroots
```

Comment:
I didn't do too much research about this, but I happened to see this: https://www.reddit.com/r/swaywm/comments/1l8f5hi/no_longer_launching_correctly_through_sddm/

would this cause issues?